### PR TITLE
Unittests: SuppressSrcTest as an update (psr3) of SuppressTest

### PR DESCRIPTION
**Pull request by boekhorstb1**

### CODE REVIEW

**Comment by midahp** [comment]
@boekhorstb1 Please name the file and the class `SupressFilterTest`

**Review by ralflang — APPROVED** [review]

File: test/Filter/SuppressSrcTest.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+class SuppressSrcTest extends TestCase
+{
+    public function setUp(): void
+    {
+        $this->filter = new SuppressFilter();
+        $this->level1 = new LogLevel(1, 'testName1');
+        $this->level2 = new LogLevel(2, 'testName2');
+        $this->level3 = new LogLevel(3, 'testName3');
+        $this->message1 = "test1";
```
Comment (ralflang) [inline test/Filter/SuppressSrcTest.php]: This should be 'single quotes'

File: test/Filter/SuppressSrcTest.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * @subpackage UnitTests
+ */
+namespace Horde\Log\Test\Filter;
+
+use \PHPUnit\Framework\TestCase;
+use Horde\Log\Filter\SuppressFilter;
+use Horde\Log\LogFilter;
+use Horde\Log\LogMessage;
+use Horde\Log\LogLevel;
+use TypeError;
```
Comment (ralflang) [inline test/Filter/SuppressSrcTest.php]: TypeError is NOT used. You may drop it.

**Comment by ralflang** [comment]
@boekhorstb1 I really like that one. Can you please address the comments from the review nevertheless before I merge it later this week?

File: test/Filter/SuppressSrcTest.php (hunk 2 above)
Comment (boekhorstb1) [reply test/Filter/SuppressSrcTest.php]: Ok I will drop TypeError in SuppressTest.php

File: test/Filter/SuppressSrcTest.php (hunk 1 above)
Comment (boekhorstb1) [reply test/Filter/SuppressSrcTest.php]: ok will change this

**Comment by midahp** [comment]
@ralflang  I would close this since https://github.com/maintaina-com/Log/pull/5 included this code and was already merged.

**Pull request closed without merging**